src/plugins/intel_cpu/src/nodes/pooling.cpp

```diff
@@ -237,6 +248,65 @@ void Pooling::getSupportedDescriptors() {
     InferenceEngine::Precision inputPrecision = getOriginalInputPrecisionAtPort(0);
     InferenceEngine::Precision outputPrecision = getOriginalOutputPrecisionAtPort(0);
 
+#if defined(OV_CPU_WITH_ACL)
+    useACL = true;
+    const auto &parentShape1 = getInputShapeAtPort(0);
+    const auto &childShape1 = getOutputShapeAtPort(0);
+    auto inShape1 = MemoryDescUtils::makeDummyShape(parentShape1);
+    if (isDynamicNode()) {
+        const auto& origDims = parentShape1.getDims();
+        const auto& origMaxDims = parentShape1.getMaxDims();
+
+        auto inDims = inShape1.getStaticDims();
+        for (size_t i = 0; i < inDims.size() - 2; i++) {
+            if (origDims[i + 2] == Shape::UNDEFINED_DIM) {
+                inDims[i + 2] = std::min<Dim>(origMaxDims[i + 2], std::max<Dim>(inDims[i + 2], poolingAttrs.kernel[i]));
+            }
+        }
+        inShape1 = Shape(inDims);
+    }
+
+    // WA: we may specify any layout here (NCHW or NHWC) since both are supported by ACL
+    arm_compute::TensorInfo srcTensorInfo = arm_compute::TensorInfo(shapeCast(MemoryDescUtils::makeDummyShape(parentShape1).getDims()),
+                                                                    1,
+                                                                    precisionToAclDataType(inputPrecision),
+                                                                    arm_compute::DataLayout::NCHW);
+    arm_compute::TensorInfo dstTensorInfo = arm_compute::TensorInfo(shapeCast(MemoryDescUtils::makeDummyShape(childShape1).getDims()),
+                                                                    1,
+                                                                    precisionToAclDataType(outputPrecision),
+                                                                    arm_compute::DataLayout::NCHW);
+    arm_compute::PoolingLayerInfo pool_info;
+    unsigned int pad_left = (poolingAttrs.data_pad_begin.size() == 2) ? poolingAttrs.data_pad_begin[1] : 0;
```

Comment:
@dmitry-gorokhov what value should be used for pad_right, kernel_w and stride_x vars if the list contains 1 element?
I put some values based on test results but I'm pretty sure I put incorrect values.

Reply:
If I remember correctly size of these attributes equal 1 only in case input tensor is 3D. As a fastet solution we can disable 3D case for ACL and put the task in backlog.
Alternatively we can reinterpret 3D shape as 4D: N C W => N C 1 W on ACL Executor level. Then kernel: [1 , kernel_w], pad [0, pad], stride [1, stride_w].

Reply:
Supported 3D case